netlify.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+  publish = "./site"
+  command = "source /opt/buildhome/python3.7/bin/activate; pip3 install mkdocs mkdocs-awesome-pages-plugin; sed -i 's/site_url: https:\/\/kubevirt.io\/docs/site_url: https:\/\/kubevirt.io\//' /opt/build/repo/mkdocs.yml; sed -i 's/docs_dir: docs/docs_dir:/' /opt/build/repo/mkdocs.yml; echo '*** BEGIN /opt/build/repo/mkdocs.yml ***'; cat /opt/build/repo/mkdocs.yml; echo '*** END /opt/build/repo/mkdocs.yml ***'; mkdocs build -f /opt/build/repo/mkdocs.yml -d /opt/build/repo/site"
```

Comment:
shouldn't this look like `make build` to invoke the same command that will be used to render the site?

Reply:
favicon is the icon the browsers display on the tab.

`docs/img/favicon.ico` is the default location for the mkdocs engine if it's not overridden by theme->icon->logo.  I guess I was tinkering here and didn't remove my mess.

Reply:
The rest of the icons in `assets/` already exist in `master`.  They were transferred over.  I don't know if someone stuck icons into content.  I didn't check.

Reply:
Notice python in Netlify lives in /opt/buildhome/python3.7?  That's a non standard path.  I'm not sure python 3.7 is even in $PATH in the Netlify image.  I have to investigate if we can merge this build command into `make build`.  I'm sure we can I just need time.  In the meantime this works.